build-logic/src/main/kotlin/org.sonarsource.iac.code-style-convention.gradle.kts

```diff
@@ -0,0 +1,23 @@
+import com.diffplug.blowdryer.Blowdryer
+
+plugins {
+  id("com.diffplug.spotless")
+}
+
+spotless {
+  encoding(Charsets.UTF_8)
+  java {
+    eclipse("4.21.0")
+      .configFile(
+        Blowdryer.immutableUrl(
+          "https://raw.githubusercontent.com/SonarSource/sonar-developer-toolset/master/eclipse/sonar-formatter.xml",
+        ),
+      )
+    licenseHeaderFile(rootProject.file("LICENSE_HEADER")).updateYearWithLatest(true)
+  }
+  kotlinGradle {
+    ktlint().setEditorConfigPath("$rootDir/.editorconfig")
```

Comment:
The file is missing

Reply:
The file actually exists, but it is indeed missing Kotlin configuration, I will complete it.
FYI I tested, if the file is missing then the gradle build failed.